packages/components/src/templates/next/components/complex/DynamicDataBanner/DynamicDataBannerClient.tsx

```diff
@@ -48,6 +42,17 @@ type DynamicDataBannerClientProps = Omit<
   errorMessageBaseParagraph?: React.ReactNode
 }
 
+const ErrorMessage = ({
+  errorMessageBaseParagraph,
```

Comment:
fyi - using `children` is recommended as a prop there because it allows you to use the default jsx nesting!

```tsx
<ErrorMessage>
  <Text> 
    asdf
  </Text>
</ErrorMessage>
```

but is more of a style thing here so no matter

Reply:
another alternative (based on how it's being used) is that we could consider a `Skeleton` api where we pass in `children` + `errorMessageBaseParagraph` and if `!!errorMessageBaseParagraph`, we show it, otherwise we render children.

benefits are that the code gets slightly easier to read because we'd have to jump back and forth atm due to the 2 ternaries